src/content/docs/products/ontrack/start-contributing.md

```diff
@@ -0,0 +1,101 @@
+
+#### Running the Dev Container
+To run the dev container, first you must get the **Docker Engine** running and open the **doubtfire-deploy** folder in a dev container using VS Code's Command Palette (ctrl / cmd + shift + p) command ```Dev Containers: Open Folder in Container...``` 
+ 
+Make sure the branch is development and the git remotes are configured properly. It should look something like this for all 3 repos (doubtfire-deploy, doubtfire-web, doubtfire-api):
+
+![Git Remotes](/ontrack/contributing-guide/gitremotes.png)
+
+**Origin** should point to the **fork that you have created** and **upstream** should point to the **Thoth Tech** repo
+
+#### Common errors during dev container setup
+
+##### 1. MySQL error in the backend
+![MySQL Backend Error](/ontrack/contributing-guide/mysqlerror.png)
+
+###### Solution
+Dev container related files are only located in the **development** branch under ***.devcontainer***. This means if you are on the wrong branch or your container doesn't recognise the git repository, it won't find the dev container files and your container won't be configured properly (your Configuring... terminal would show an error like shown below e.g. post_start.sh was not found.)
+
+![Post Start Error](/ontrack/contributing-guide/poststarterror.png)
+
+On Windows, your Docker workspace won't recognise the git repository if you don't configure git to mark the workspace as a safe directory. If you do a ```git fetch``` like shown below, git will alert you of this and tell you which command to run to fix it. Copy paste that command and run it. After that, you'll see the branch name in parenthesis as shown below in red. Do this for all 3 repos to mark each directory as safe for git operations.
+
+![Git Fetch Example](/ontrack/contributing-guide/gitfetchexample.png)
+
+**Red** means there is a problem with your branch. Ideally the branch name should be in **green**. To fix this, run ``git reset --hard upstream/development`` for each repo. This is assuming you have fetched the latest changes for each repo using ```git fetch``` and you are on the development branch. Run ```git status``` to make sure everything's up to date and there are no pending changes.
+
+Then without closing the remote connection to the container, rebuild the container using the Command Palette command (ctrl / cmd + shift + p) ```Dev Containers: Rebuild Container```. 
+
+##### 2. Docker container related errors
+Sometimes if you are having any weird Docker container related problems such as shown below, you might need to delete all **Containers, Images, and Volumes** related to **doubtfire-lms/formatif** from inside the **Docker app** and restart the **Docker Engine**.
```

Comment:
Please also add a reminder to check if any updates are pending for Docker. Not updating the app might disrupt it's behaviour as well.